jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/profiles/ProfileCredentialProviderFactoryTest.kt

```diff
@@ -19,6 +19,7 @@ import org.junit.After
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
+import org.junit.jupiter.api.assertThrows
```

Comment:
we're using assertj everywhere else, so prefer that unless you have a good reason